config/environments/test.rb

```diff
@@ -49,7 +49,7 @@
   # Raise exceptions for disallowed deprecations.
   config.active_support.disallowed_deprecation = :raise
 
-  # Tell Active Support which deprecation messages to disallow.
+  # Tell Active Support which deprecation notes to disallow.
```

Comment:
WHY?

Reply:
when we change the table from message to notes, all in app where we were used 'message / messages' word replaced with note / notes.